src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/player/ModuleRegen.kt

```diff
@@ -40,12 +40,17 @@ object ModuleRegen : Module("Regen", Category.PLAYER) {
     }
 
     val repeatable = repeatable {
+
         if (resetTimer) {
             mc.timer.timerSpeed = 1F
             resetTimer = false
         }
 
-        if ((!noAir && player.isOnGround) && !player.abilities.creativeMode && player.health >= 0 && player.health < health) {
+        if ((!noAir && player.isOnGround) && !player.abilities.creativeMode && player.health < health) {
```

Comment:
`player.health >= 0` implies that he is alive

Reply:
Does it really though? >= means higher or equal to a number, it should be just `>`.